pmd-apex/src/test/resources/net/sourceforge/pmd/lang/apex/rule/security/xml/ApexCRUDViolation.xml

```diff
@@ -376,6 +376,48 @@ public class Foo {
         ]]></code>
     </test-code>
 
+    <test-code>
+        <description>Flagged as Critical With No Mode</description>
+        <expected-problems>1</expected-problems>
+        <code><![CDATA[
+public class User {
+    public void coverAllCasesWithTest() {
+        Contact c;
+		c = [SELECT Name FROM Contact];
+    }
+}
+        ]]></code>
+    </test-code>
+
+    <test-code>
+        <description>User Mode</description>
+        <expected-problems>0</expected-problems>
+        <code><![CDATA[
+public class User {
+    public void coverAllCasesWithTest() {
+        Contact c;
+		c = [SELECT Name FROM Contact WITH USER_MODE];
+    }
+}
+        ]]></code>
+    </test-code>
+
+    <test-code>
+        <description>System Mode, gives warning because it ignores CRUD but explicitly</description>
+        <expected-problems>1</expected-problems>
```

Comment:
As @adangel said this should also reveal no issue as explicit usage is not an issue.

Reply:
Changed that now back to not report a violation, if system mode is explicitly used.